dace/sdfg.py

```diff
@@ -310,9 +310,6 @@ def from_json(cls, json_obj, context_info={'sdfg': None}):
 
         return ret
 
-        # Counter to make it easy to create temp transients
-        self._temp_transients = 0
```

Comment:
This code still exists in `__init__`. It was duplicated as a result of a merge conflict and is unreachable.